.github/workflows/aws-upload-layer-on-release.yml

```diff
@@ -63,5 +75,20 @@ jobs:
           
           # copy the file to S3 and install it in lambda layers
           aws s3 cp $release_id $release_bucket_uri
-          # fixme: update compatible runtimes
-          aws lambda publish-layer-version --layer-name $release_layer  --content S3Bucket=$release_bucket,S3Key=$release_file_key --compatible-runtimes  python3.8
+          aws lambda publish-layer-version --layer-name $release_layer  --content S3Bucket=$release_bucket,S3Key=$release_file_key  --license-info 'Apache-2.0' --compatible-runtimes  python3.8 > layer.json
+      
+      - name: Extract layer info
+        run: |
+          # see https://docs.github.com/en/actions/learn-github-actions/workflow-commands-for-github-actions#setting-an-environment-variable
+          # without version
+          echo "LAYER_ARN=$(cat layer.json | jq -r '.LayerArn')" >> $GITHUB_ENV
+          # arn with version
+          echo "LAYER_VERSION_ARN=$(cat layer.json| jq -r '.LayerVersionArn')" >> $GITHUB_ENV
+          # just the version number, without ARN
+          echo "LAYER_VERSION_NUMBER=$(cat layer.json| jq -r '.Version')" >> $GITHUB_ENV
+
+      - name: Set tags and permissions for lambda layer
+        run: |
+          aws resourcegroupstaggingapi tag-resources --resource-arn-list "$LAYER_VERSION_ARN" --tags '{"VantaOwner":"customer services","VantaNonProd":"true","VantaDescription":"AWS Lambda Layer built from public kensu-py repo","VantaContainsUserData":"false","VantaNoAlert":"created only from public kensuio-oss kensu-py repo"}'
```

Comment:
P.S. still not sure if this is the right command for setting tags. my user had no permission to call this operation, so might need to comment it out for now